package.json

```diff
@@ -9,6 +9,7 @@
     "node": "^4 || ^5"
   },
   "devDependencies": {
+    "SMS": "file:./apps/sms",
```

Comment:
When we do ```npm install xxx --save```, npm will rearrange dependencies automatically. So I prefer to keep this change in this patch.